Analyses/src/CaloExample_module.cc

```diff
@@ -365,7 +365,12 @@ namespace mu2e {
            }
```

Comment:
Analyses is a big mix of current and obsolete code.  I suggest creating CaloDiag for this and similar modules.

Reply:
Ok